README.md

```diff
@@ -76,16 +76,25 @@ slots are determined by the recurring weekly pattern.
 
 #### `Visit`
 
-This is the main table in the application, and contains all the essential data
-for a visit: the prison, prisoner details, and primary visitor's details and
-contact information.
+This is the main table in the application, and contains the essential data for
```

Comment:
I like that you guys have notes like this, just hope that they are kept in sync.